pkg/processor/build/runtime/python/docker/onbuild/Dockerfile

```diff
@@ -60,7 +61,7 @@ RUN pip download \
         --requirement /requirements/python3_10.txt
 
 # Supplies python 3.11 & common wheels
-FROM gcr.io/iguazio/python:3.11 as python311-builder
+FROM $NUCLIO_PYTHON_BASE_IMAGE_NAME:3.11 as python311-builder
```

Comment:
Minor - If we allow passing the image here, probably we should allow passing tag as well?

Reply:
This docker file has different stages for all of the Nuclio supported python versions.
If we allow passing the tags, how will we know how many stages are there? (We can't do a for-loop)

Reply:
@TomerShor then alright, we just need to make sure that if we pass custom image, this tag exist 👍